tests/integration/test_remove_relation.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
```

Comment:
Please improve the PR description. Here you want to focus on why you made the change. Please remove the line `It is part of onboarding discussion`.